### ISSUE TITLE
Replace hacky `fancy_copy.py` script with CMake copy

### ISSUE THREAD

**Issue by nbolton** [deskflow/deskflow#7754]
### Feature Request

The `fancy_copy.py` script was created to work around problems with Windows and file in use running process lock problems related to working with the daemon and Windows services. We build to a temp dir on Windows so that we can guarantee the build always works, even if processes are running (such as the Windows service). It is not practical to shut down the Windows service every time you want to compile.

There is probably a way of doing this in CMake, so that we can get rid of `fancy_copy.py`. We need to mimic what `fancy_copy.py` does, which is copy the files and ignore each file copy if the file is in use by Windows, but show a warning. There is also a related `daemon.py` script which installs and reinstalls the Windows service daemon, and that uses the same copy logic in ``fancy_copy.py`.

TL;DR: Developing with Windows services (and Windows in general) is a horrible experience and `fancy_copy.py` works around this problem. It would be better if CMake could do this so we don't have to use Python.

Idea: Make this funky copy logic a CMake option, so that by default it just builds normally to the build dir. A developer can optionally turn on the "safe" build/copy feature.
